tests/contrib/httplib/test_httplib.py

```diff
@@ -353,7 +353,7 @@ def test_httplib_request_and_response_headers(self):
             self.assertEqual(s.get_tag('http.response.headers.access_control_allow_origin'), None)
 
         # Enabled when configured
-        with self.override_config('hhtplib', {}):
+        with self.override_config('httplib', {}):
```

Comment:
Is this doing anything then? If the test work, I imagine this line is useless, no?

Reply:
I think it's here to avoid side-effects.

Reply:
Yeah I think it's here as a "just-in-case" the config had been changed elsewhere prior to the test. I think it's fine to leave it in in case such a change does occur in the future.